extension-config.json

```diff
@@ -3,28 +3,34 @@
   "id": "@shopgate-project/product-image-badges",
   "components": [
     {
-      "id": "ComponentsProductImageAfter",
+      "id": "ProductItemPriceBefore",
       "type": "portals",
-      "path": "frontend/portals/ComponentProductImageAfter",
-      "target": "component.product-image.after"
+      "path": "frontend/portals/ProductItemPriceBefore",
+      "target": "product-item.price.before"
+    },
+    {
+      "id": "ComponentProductImageBefore",
+      "type": "portals",
+      "path": "frontend/portals/ComponentProductImageBefore",
+      "target": "component.product-image.before"
     }
   ],
   "configuration": {
     "badgeMap": {
       "type": "admin",
       "default": [
         {
-          "tags": [
+          "triggerTags": [
```

Comment:
I think we should remove the defaults for the final commit.

Reply:
Removed defaults and updated Readme